torch/testing/_internal/common_methods_invocations.py

```diff
@@ -148,8 +165,11 @@ def __init__(self,
         else:
             self.autodiff_nonfusible_nodes = autodiff_nonfusible_nodes
         self.supports_sparse = supports_sparse
-
-
+        self.sparse_op_info = sparse_op_info if sparse_op_info is not None else {}
+        self.operator_variant = getattr(operator, name, None)
```

Comment:
Adding operator variants is interesting but we should probably separate it into its own PR